docs/lessons/02-recap.md

```diff
@@ -0,0 +1,88 @@
+# 01 - Introduction to Intermediate Java
+<Teacher name="Flo"></Teacher>
+
+## Goals
+
+- Understand java and javac
+- Understand what bytecode is
+- Being able to run javac on the commandline and find the generated files
+- Do some basic Java exercises 🤩
+
+
+## Slides
+
+<GoogleSlides src="https://docs.google.com/presentation/d/1CG5d6teQjFnKcJ_d8YvcTvSOPcUEeO8D-U3Oq7nvqRw/edit#slide=id.p"></GoogleSlides>
```

Comment:
Not sure if this works - usually the link in the `GoogleSlides` tag should end with `/embed`.

Reply:
Makes sense - I just posted the link there to my slides :-)